{
  addExceptionExpectation();
  doThrow(expectedExceptionCause).when(connection).rollback();
  tx.rollback();
}
